src/mysql_shell.py

```diff
@@ -52,6 +52,7 @@ def _run_commands(self, commands: typing.List[str]) -> str:
         commands.insert(
             0, f"shell.connect('{self.username}:{self._password}@{self._host}:{self._port}')"
         )
+        commands.append("shell.disconnect()")
```

Comment:
@carlcsaposs-canonical were you able to query the database for the opened connection?
My feeling is that, when running in batch mode (`--file`) the connection closes (or it should) when the file is over.